TCAT/Model/BusLocation.swift

```diff
@@ -32,16 +32,42 @@ class BusLocation: NSObject {
     var lastUpdated: Date
     var latitude: Double
     var longitude: Double
-    var name: Int
+    var name: String?
     var opStatus: String
-    var routeID: String
+    var routeID: Int?
     var runID: Int
     var speed: Int
-    var tripID: String
+    var tripID: Int?
     var vehicleID: Int
-    
+
     private var _iconView: UIView?
-    
+
+    private let noDataString = "noData"
+    private let validDataString = "validData"
+    private let invalidDataString = "invalidData"
+
+    private enum CodingKeys: String, CodingKey {
+        case dataType = "case"
+        case destination
```

Comment:
Just confirming, do you need to list all of these if you only have to change one CodingKey?

Reply:
Yea unfortunately